packages/asc-ui/src/components/Typography/TypographyStyle.ts

```diff
@@ -5,12 +5,16 @@ import { Theme } from '../../types'
 export type Props = {
   gutterBottom?: number
   element?: Variant
-  color?: Theme.ColorType
   fontSize?: number
   styleAs?: keyof Theme.TypographyElements
   as?: any
   forwardedAs?: any
   strong?: boolean
+  /**
+   * Deprecated: Use your own custom style rules
+   * @deprecated
+   */
```

Comment:
The text you put after `@deprecated` will be used by the IDE. So I would change this to:

```suggestion
  /**
   * @deprecated Use your own custom style rules
   */
```

Reply:
Sure!